mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
re: `$(INSTALL_LIB) $(ROSLYN_FILES_FOR_MONO) $(TARGET_DIR)`

This is existing bit that copies csc.exe and related files to `lib/mono/4.5`.

re: `.. copy only real msbuild ..`

Sounds good to me. The `*Scripting*` assemblies should be wherever `csi.exe` is, so I will install it to `lib/mono/4.5` too.